docs/getting_started/conventions.md

```diff
@@ -0,0 +1,168 @@
+# Conventions for Mapbender
```

Comment:
I guess we have to discuss the conventions

Reply:
will be done for the next release